From f338b5bc81aeb31f1c0686493abf08a14025faf4 Mon Sep 17 00:00:00 2001
From: Xi Ruoyao <xry111@xry111.site>
Date: Thu, 4 Dec 2025 18:15:00 +0800
Subject: [PATCH 20/63] loonggpu-bridge: make is_resolution_valid static

Now it's only used in the TU where we define it.

Signed-off-by: Xi Ruoyao <xry111@xry111.site>
---
 loonggpu-bridge/bridge_phy.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/loonggpu-bridge/bridge_phy.c b/loonggpu-bridge/bridge_phy.c
index 45fcfd0..cd9b120 100644
--- a/loonggpu-bridge/bridge_phy.c
+++ b/loonggpu-bridge/bridge_phy.c
@@ -99,7 +99,7 @@ static int bridge_phy_connector_get_modes(struct drm_connector *connector)
 	return count;
 }
 
-bool is_resolution_valid(struct panel_resource *panel_resource, struct drm_display_mode *mode)
+static bool is_resolution_valid(struct panel_resource *panel_resource, struct drm_display_mode *mode)
 {
 	u32 vrefresh = 0;
 	u32 index;
-- 
2.52.0

